### PR TITLE
fix(v1): self-host user images

### PR DESCRIPTION
**Pull request by leoigel**
## Motivation

(Write your motivation here.)
 Improve reliability
### Have you read the [Contributing Guidelines on pull requests](https://github.com/facebook/docusaurus/blob/master/CONTRIBUTING.md#pull-requests)?
Yes
(Write your answer here.)
In https://docusaurus.io/en/users i saw one image not loaded.
## Test Plan

(Write your test plan here. If you changed any code, please provide us with clear instructions on how you verified your changes work. Bonus points for screenshots and videos!)
I replaced all external  url to self-host all images.

## Related PRs

(If this PR adds or changes functionality, please take some time to update the docs at https://github.com/facebook/docusaurus, and link to your PR here.)

### CODE REVIEW

**Comment by leoigel** [comment]
Some thing is missing here? the pr was approved?

File: website-1.x/data/users.js [hunk 1]
```diff
@@ -506,14 +506,14 @@ module.exports = [
...
-    image: 'https://pnpm.js.org/img/pnpm.svg',
+    image: '/img/users/pnpm.svg',
     infoLink: 'https://pnpm.js.org/',
     fbOpenSource: false,
     pinned: false,
   },
   {
     caption: 'Polymath Network',
-    image: 'https://developers.polymath.network/img/text.svg',
+    image: '/img/users/polymatch-logo.png',
```
Comment (slorber) [inline website-1.x/data/users.js:516]: how comes it's not a png?

File: website-1.x/data/users.js [hunk 2]
```diff
@@ -737,7 +737,7 @@ module.exports = [
   },
   {
     caption: 'smash.gg',
-    image: 'https://imgur.com/eBFBDei.png',
+    image: '/img/users/eBFBDei.png',
```
Comment (slorber) [inline website-1.x/data/users.js:740]: rename it please

File: website-1.x/data/users.js [hunk 3]
```diff
@@ -317,7 +317,7 @@ module.exports = [
   },
   {
     caption: 'Home Assistant',
-    image: 'https://developers.home-assistant.io/img/logo-responsive.svg',
+    image: '/img/logo-responsive.svg',
```
Comment (slorber) [inline website-1.x/data/users.js:320]: rename it + fix the link (missing /users in path)

File: website-1.x/data/users.js (hunk 1 above)
Comment (leoigel) [reply website-1.x/data/users.js:516]: I took png image from official website https://polymath.network/

File: website-1.x/core/Showcase.js [hunk 4]
```diff
@@ -21,13 +21,23 @@ UserLink.propTypes = {
...
 
-const Showcase = ({users}) => (
-  <div className="showcase">
-    {users.map((user) => (
-      <UserLink key={user.infoLink} {...user} />
-    ))}
-  </div>
-);
+const Showcase = ({users}) => {
+  users.forEach((user) => {
```
Comment (slorber) [inline website-1.x/core/Showcase.js:25]: this check should be outside of the render method, at the top level

File: website-1.x/core/Showcase.js [hunk 5]
```diff
@@ -7,7 +7,15 @@
...
 const PropTypes = require('prop-types');
+const users = require('../data/users');
 
+users.forEach((user) => {
+  if (!user.image.startsWith('/img/users')) {
+    throw new Error(
+      `User image should be self-hosted in /img/users folder. This was not the case for ${user.image}`,
+    );
+  }
+});
```
Comment (slorber) [inline website-1.x/core/Showcase.js:18]:
```suggestion
});

```

Comment (slorber) [reply website-1.x/core/Showcase.js:18]: just adding a space here

**Comment by slorber** [comment]
thanks!

**Pull request merged**